libvast/src/plugin.cpp

```diff
@@ -216,7 +216,7 @@ caf::error initialize(caf::actor_system_config& cfg) {
     }
     // Second, try to read the configuration from the plugin-specific
     // configuration files at <config-dir>/plugin/<plugin-name>.yaml.
-    for (auto&& config_dir : system::config_dirs(cfg)) {
+    for (auto&& config_dir : system::config_dirs()) {
```

Comment:
@dominiklohmann or @tobim: please note that I switched `config_dirs` into a singleton in this commit to begin breaking down the tight coupling into `actor_system` where possible.

Reply:
I've reverted this change and restored the original function signature of `config_dirs`. It turns we use this pattern of hauling things through the `actor_system_config` instance more pervasively than I initially thought.